test/google_sign_in_mocks_test.dart

```diff
@@ -2,14 +2,28 @@ import 'package:google_sign_in_mocks/google_sign_in_mocks.dart';
 import 'package:test/test.dart';
 
 void main() {
+  late MockGoogleSignIn googleSignIn;
+  setUpAll((){
```

Comment:
Can you replace this by `setUp`? We'd like each test to be independent.

Reply:
Okay, done.